river/evaluate/tracks.py

```diff
@@ -0,0 +1,67 @@
+import typing
+
+from river import datasets, metrics
+from river.evaluate.progressive_validation import _progressive_validation
+
+__all__ = ["load_binary_clf_tracks"]
+
+
+class Track:
+    """A track evaluate a model's performance.
+
+    The following metrics are recorded:
+
+    - FLOPS: floating point operations per second.
```

Comment:
That's going to be a gem! I can't wait to incorporate that in my PhD's experiments :D

Reply:
Yes I agree! There's [this](https://github.com/firedrakeproject/PyPAPI) Python package but I haven't managed to make it work on MacOS.